src/cast_core/test/CrudActorYamlTests.yml

```diff
@@ -716,3 +716,285 @@ Tests:
             Pipeline: {$group: {_id: "$a", sum: {$sum: "$b"}}}
     Error: |
       .*'Pipeline' must be an array.*
+
+  - Description: Test that a 'find' command applies the specified projection.
+    Operations:
+      - OperationName: find
+        OperationCommand:
+          Filter: {a: 100}
+          Options:
+            Projection: {_id: 0, b: 1}
```

Comment:
[optional] We don't have to do this if it's tricky, but is there an easy way to test that the `DocumentGenerator` is kicking in correctly for the `Projection` option? Maybe we could use a simple `^Inc` generator or the like in order to show that we can generate the first integer in a sequence?